coordinates/w64h64.json.example

```diff
@@ -105,7 +105,15 @@
   },
   "batter_count": {
     "x": 46,
-    "y": 46
+    "y": 46,
+    "nohit": {
+      "x": 46,
+      "y": 46
+    },
+    "perfect_game": {
+      "x": 46,
+      "y": 46
+    }
```

Comment:
I don't believe this is used anymore, but it's in most of the coordinates examples, so I added here.

Reply:
My custom layout uses this feature pretty extensively but not in any of the places the example does